torchvision/prototype/models/depth/stereo/crestereo.py

```diff
@@ -1055,11 +1057,145 @@ def _crestereo(
     return model
 
 
+_COMMON_META = {
+    "min_size": (384, 512),
+}
+
+
 class CREStereo_Base_Weights(WeightsEnum):
-    pass
+    """The metrics reported here are as follows.
+
+    ``mae`` is the "mean-average-error" and indicates how far (in pixels) the
+    predicted disparity is from its true value (equivalent to ``epe``). This is averaged over all pixels
+    of all images. ``1px``, ``3px``, ``5px`` and indicate the percentage of pixels that have a lower
+    error than that of the ground truth. ``relepe`` is the "relative-end-point-error" and is the
+    average ``epe`` divided by the average ground truth disparity. ``f1`` corresponds to the average of pixels whose epe
+    is either >3px, or whom's ``relepe`` is higher than 0.05 (therefore lower is better).
```

Comment:
@TeodorPoncu Just curious, where this definition of `f1` comes from? It seems quite custom to me.
Also if I understand correctly this is something like the average value of the disparity that is considered "bad"?

Reply:
It's similar to the `f1` used in Optical Flow, main difference is the norm2 of the flow is just norm1 since the second channel is always 0.

Reply:
Oh, I think there is a typo here. In optical flow it is called `fl-all` notice it use small `L` rather than `1`.
see https://github.com/pytorch/vision/blob/main/torchvision/models/optical_flow/raft.py#L682
From what I see on [KITTI](https://www.cvlibs.net/datasets/kitti/eval_scene_flow.php?benchmark=flow) it is meant for optical flow outlier.

Lets fix the naming (probably need to fix the one in reference as well) so people dont mixed this with the standard `f1`.